app/vmalert/config/config.go

```diff
@@ -208,8 +208,7 @@ var cLogger = &log.Logger{}
 func ParseSilent(pathPatterns []string, validateTplFn ValidateTplFn, validateExpressions bool) ([]Group, error) {
 	cLogger.Suppress(true)
 	defer cLogger.Suppress(false)
-
-	files, err := readFromFS(pathPatterns)
+	files, err := readFromFSOrHTTP(pathPatterns)
```

Comment:
I think it would be more convenient to introduce a new package `app/vmalert/config/url`. In the similar way to `fslocal`.

Reply:
Yeah, I shouldn't read http path in fs.go, separated them in the new commit, is that ok?